templates/templates/_navigation-community-h.html

```diff
@@ -7,7 +7,7 @@ <h4 class="p-heading--four is-dense u-hide--small">Learn</h4>
           <h4 class="p-muted-heading u-hide--medium u-hide--large">Learn</h4>
           <p class="p-p--small u-hide--small">Tutorials cover a wide range of topics. Learn or teach something new to Ubuntu users.</p>
           <ul class="p-text-list--small is-bordered">
-            <li class="p-list__item"><a href="http://tutorials.ubuntu.com/">Ubuntu Tutorials</a></li>
+            <li class="p-list__item"><a href="https://www.ubuntu.com/tutorials/">Ubuntu Tutorials</a></li>
```

Comment:
```suggestion
            <li class="p-list__item"><a href="/tutorials">Ubuntu Tutorials</a></li>
```
we don't need the domain